src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -206,4 +209,48 @@ private function decidePriority(TokenInterface $token, array $attributes, $objec
 
         return $this->allowIfAllAbstainDecisions;
     }
+
+    private function getVoters(array $attributes, $object = null): iterable
+    {
+        $keyAttributes = [];
+        foreach ($attributes as $attribute) {
+            $keyAttributes[] = \is_string($attribute) ? $attribute : null;
+        }
+        // use `get_class` to handle anonymous classes
+        $keyObject = \is_object($object) ? \get_class($object) : get_debug_type($object);
```

Comment:
```suggestion
        $keyObject = get_debug_type($object);
```

It's the same, right?

Reply:
not really https://3v4l.org/YM635

Reply:
Good catch, sorry for the noise

Reply:
see comment line 219 ;)